devise_saml_authenticatable.gemspec

```
@@ -19,5 +19,5 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.6.0"
 
   gem.add_dependency("devise","> 2.0.0")
-  gem.add_dependency("ruby-saml","~> 1.7")
+  gem.add_dependency("ruby-saml","~> 1.17.0")
```

I don’t think we need to limit to only 1.17.x, any minor version higher should be fine too. 

```suggestion
  gem.add_dependency("ruby-saml","~> 1.17")
```

---

Thank you for the feedback! I just pushed a fixup containing the change you suggested. 

Please let me know when I can change the commit messages .

---

It should be fine to rebase if you’d like!

---

Rebased!